packages/core/content-type-builder/admin/src/utils/parseDateValue.ts

```diff
@@ -0,0 +1,18 @@
+import { Schema } from '@strapi/types';
+
+export const parseDateValue = <TAttribute extends Schema.Attribute.AnyAttribute>(
+  value: Schema.Attribute.Value<TAttribute>
+): Date | undefined => {
+  if (value instanceof Date) {
```

Comment:
```
  if (value instanceof Date && isValidate(value)) {
```
you can do this here instead and then just return the value, looks a little neater not having to type `undefined`